lib/abide_dev_utils/xccdf.rb

```diff
@@ -10,6 +10,17 @@
 module AbideDevUtils
   # Contains modules and classes for working with XCCDF files
   module XCCDF
+    # Generate map for CEM
+    def self.gen_map(xccdf_file, opts)
+      type = opts.fetch(:type, 'cis')
+      case type.downcase
+      when 'cis'
+        new_map = Benchmark.new(xccdf_file).gen_map(**opts)
+      else
+        AbideDevUtils::Output.simple("XCCDF type #{type} is unsupported!")
```

Comment:
This should be raised as an exception. We shouldn't really ever handle errors by just printing to the console, especially from library code.